front/src/applications/operationalStudies/views/Project.js

```diff
@@ -17,14 +18,16 @@ import { useSelector, useDispatch } from 'react-redux';
 import { get } from 'common/requests';
 import { setSuccess } from 'reducers/main';
 import FilterTextField from 'applications/operationalStudies/components/FilterTextField';
-import AddOrEditStudyModal from '../components/Study/AddOrEditStudyModal';
 import { PROJECTS_URI, STUDIES_URI } from '../components/operationalStudiesConsts';
 import AddOrEditProjectModal from '../components/Project/AddOrEditProjectModal';
 import BreadCrumbs from '../components/BreadCrumbs';
 
 function displayStudiesList(studiesList, setFilterChips) {
   return studiesList ? (
     <div className="row no-gutters">
+      <div className="col-xl-6" key={nextId()}>
```

Comment:
`<div className="col-xl-4 col-lg-6" key={nextId()}>`

to preserve layout between different views

Reply:
Done with 3d998f9